modules/s3db/hrm.py

```diff
@@ -193,7 +195,9 @@ def model(self):
             sortby = "name",
             comment = S3AddResourceLink(c="vol" if group == "volunteer" else "hrm",
                                         f="department",
-                                        label=label_create),
+                                        label=label_create,
+                                        title=label,
```

Comment:
title = label_create ?
I don't see the point of creating a new label variable that tells the exact same thing, i.e, "Create Department" ;)